WebApplication/Controllers/MessageBrokerController.cs

```diff
@@ -19,10 +21,40 @@ public MessageBrokerController(Serilog.ILogger logger)
             _logger = logger;
         }
 
-        [HttpGet]
-        public string SendMessage()
+        public string SendLog(string key, string message)
         {
-            return EmitLogTopic.Send(_logger);
+            return SendMessage("log", key, message);
+        }
+        private string SendMessage(string exchangeName, string key, string message)
+        {
+            var exchange = new TopicExchange();
```

Comment:
must be passed from constructor using DI https://metanit.com/sharp/aspnet5/6.1.php